rllib/agents/trainer.py

```diff
@@ -197,6 +197,8 @@
     # policy, even if this is a stochastic one. Setting "explore=False" here
     # will result in the evaluation workers not using this optimal policy!
     "evaluation_config": {
+        # Set `in_evaluation` flag to True, in case a Trainer needs to know.
+        "in_evaluation": True,
```

Comment:
Add a comment that this should not be set by the user.

Reply:
Ok. Done.